### PR TITLE
Initial commit of wsba-coordinator-completion-simple

### PR DESCRIPTION
**Pull request by npfr**
Here's the final Web service transactions quickstart. Same applies, take a look, if you want it included, you are welcome to it.

### CODE REVIEW

**Comment by sgilda** [comment]
Thanks Paul. I'll try to get to this tomorrow or Monday.

File: dist/src/main/assembly/README.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ contains tutorials for:
 * `servlet-async` - CDI + Asynchronous Servlet + Asynchronous EJB 
 * `servlet-filterlistener` - Servlet Filter and Listener
 * `ejb-remote` - shows how to access EJBs remotely (EJB + JNDI)
-
+* `wsba-coordinator-completion-simple` - Simple WS-BA JAX-WS Web service with coordinator driven completion
```
Comment (pmuir) [inline dist/src/main/assembly/README.md]: We actually just moved this list to /README.md :-( on Friday.

**Comment by pmuir** [comment]
Looks good, only comment is that the formatting is wrong, as with the other quickstart.

**Comment by npfr** [comment]
These comments should now be addressed.

**Comment by npfr** [comment]
Squashed

**Comment by sgilda** [comment]
Waiting for Paul's review based on his comment in pull/38, "I'll also take a look at some of the other quickstarts as I'm sure they will have hit the same issue".

**Comment by npfr** [comment]
I've had a stab at creating some "expected output". Let me know what you think. If it's acceptable, I'll apply the same pattern to the other two XTS quickstarts.

**Comment by sgilda** [comment]
Paul, I'm stuck on the instruction for starting the server. You say to run:
 $JBOSS_HOME/bin/standalone.sh --server-config=../../docs/examples/configs/standalone-xts.xml | egrep "started|stdout"
I think you need to specify where to navigate to run this command. When I start my server, I normally navigate to jboss-eap-6.0-DR10/bin and run ./standalone.sh <arguments>.
I think there may be one too many level of indirection in the ../../docs path? However, even when I change it to this, it doesn't start: 
bin $ ./standalone.sh --server-config=../docs/examples/configs/standalone-xts.xml | egrep "started|stdout"
Any idea what I'm doing wrong?

**Comment by sgilda** [comment]
BTW, I like that you show what results to expect!

**Comment by npfr** [comment]
Hi Sande,

I'm not sure what the problem is. Providing $JBOSS_HOME is set to the root of your JBossAS installation, you should be able to run that command from anywhere on your filesystem. For example, take a look at this console log where I run it from /tmp:

```
[paul@wiggum opt]$ cd /tmp
[paul@wiggum tmp]$ export JBOSS_HOME=/opt/jboss-eap-6.0.DR10b/
[paul@wiggum tmp]$ $JBOSS_HOME/bin/standalone.sh --server-config=../../docs/examples/configs/standalone-xts.xml 
=========================================================================
JBoss Bootstrap Environment
JBOSS_HOME: /opt/jboss-eap-6.0.DR10b/
JAVA: java
JAVA_OPTS: -server -Xms64m -Xmx512m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true -Dorg.jboss.resolver.warning=true -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000 -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true
=========================================================================
10:08:07,093 INFO  [org.jboss.modules] JBoss Modules version 1.1.0.CR6-redhat-1
10:08:07,530 INFO  [org.jboss.msc] JBoss MSC version 1.0.1.GA-redhat-1
10:08:07,583 INFO  [org.jboss.as] JBoss EAP 6.0.0.Alpha2 (AS 7.1.0.CR1-redhat-1) starting
...
10:08:16,046 ERROR [org.jboss.as] (Controller Boot Thread) JBoss EAP 6.0.0.Alpha2 (AS 7.1.0.CR1-redhat-1) started (with errors) in 9196ms - Started 145 of 220 services (7 services failed or missing dependencies, 66 services are passive or on-demand)
```

The location of the configuration file (standalone-xts.xml) is resolved relative to the "$JBOSS_HOME/standalone/configuration/" directory, so the "../.." is required to navigate back up to "$JBOSS_HOME". I agree, this is a pain, but it's required as XTS, currently, takes too long to boot, and can not be included in the default configuration.

I'll add a note to the readme, explaining why the non-default configuration is required.

Can you double check that you're doing the same as what i have done in the console output above?

Thanks,

Paul.

**Comment by sgilda** [comment]
Hi Paul,
I see! In AS7/EAP6, we no longer define a JBOSS_HOME environment variable. When I read your instruction to start the server, I thought you were using JBOSS_HOME as be a replaceable value like we use in the documentation. 
If your quickstart depends on the existence of the JBOSS_HOME variable, I think it needs to be explicitly stated in the README file with the example export statement, as you did above. Sorry about that.
Thanks, Sande

**Comment by npfr** [comment]
Sande,

I've clarified the startup instructions. Does this make more sense now?

Also, are you happy with the way I have done the expected output? If so I'll go ahead and apply the same pattern to the other two quickstarts.

**Comment by sgilda** [comment]
Hi Paul,
I'm trying your new startup instructions. Should I be seeing errors when I start the server? Can you see what I'm doing wrong? 

I navigate to /home/sgilda/tools/jboss-eap-6.0-DR10
Then I copy, paste, and run your command from your readme:
     ./bin/standalone.sh --server-config=../../docs/examples/configs/standalone-xts.xml | egrep "started|stdout"

This is the console output:

~ $ cd tools/jboss-eap-6.0-DR10/
jboss-eap-6.0-DR10 $ pwd
/home/sgilda/tools/jboss-eap-6.0-DR10
jboss-eap-6.0-DR10 $ ./bin/standalone.sh --server-config=../../docs/examples/configs/standalone-xts.xml | egrep "started|stdout"
08:56:55,296 ERROR [org.jboss.as](Controller Boot Thread) JBoss EAP 6.0.0.Alpha2 (AS 7.1.0.CR1-redhat-1) started (with errors) in 9991ms - Started 163 of 240 services (7 services failed or missing dependencies, 68 services are passive or on-demand)

I love the expected output section! I think I commented on that above. See the comment from 2 days ago (2012-0-13 11:00:34). :-)
Thanks, Sande

**Comment by sgilda** [comment]
Hi Paul,
Much better! My server starts and I'm seeing the expected output in the server console. Very nice and the README is very clear.
We've been trying to fix inconsistencies in the quickstarts so that the automated scripts to replace version numbers will work when we move them to EAP. Could you please make the following changes in all of your quickstarts?
- In your pom.xml file:
  *\* Change version 7.1.0-SNAPSHOT to 7.0.2-SNAPSHOT
  *\* Change org.jboss.as.plugins version from 7.1.0.CR1b to 7.1.0.Beta1b
  *\* Change the arq-jbossas\* versions from 7.1.0.CR1b to 7.1.0.Beta1b
  *\* There is a problem with with the repositories section (see https://github.com/jbossas/quickstart/issues/57), but it can't be removed until the artifacts are sync'd to maven central
- Can you rename the readme.md file to uppercase README.md so they're consistent across all quickstarts?  

Thanks, Sande

**Comment by npfr** [comment]
No problem, I'll get onto that today or first thing tomorrow.

**Comment by npfr** [comment]
Am I right in thinking that there is nothing to do for:

```
** There is a problem with with the repositories section (see https://github.com/jbossas/quickstart/issues/57), but it can't be removed until the artifacts are sync'd to maven central
```

**Comment by pmuir** [comment]
You are Paul, that is on my plate :-)

**Comment by npfr** [comment]
@sgilda I've fixed the inconsistencies.

**Comment by sgilda** [comment]
Looks great. Thanks Paul!

**Comment by sgilda** [comment]
I have to run out for a bit. I'm merge this in later today.

**Comment by sgilda** [comment]
I am getting a conflict on the dist/src/main/assembly/README.html and dist/src/main/assembly/README.md files. 
It appears this one is missing the \* `cmt` - shows how to use container managed transactions. However, the list doesn't appear to be up-to-date anyway as there's no bmt either.
Pete, should the list be kept up here? Is it OK just to merge this as is with an incomplete list?

**Comment by sgilda** [comment]
Never mind. It has the wsba-coordinator-completion-simple added as well. I will just clean it up and add all the missing quickstarts while I'm in here.

**Comment by sgilda** [comment]
Pete, I modified the dist/src/main/assembly/README.\* files to clean up the conflicts.
I pushed the merge to my own repo (https://github.com/sgilda/quickstart) for you to check before I push to jbossas.
When I look at the commit, it appears to be OK to me, but I'd feel better if you checked it out since there were conflicts. I'll send email as well.

**Comment by pmuir** [comment]
Sande, if you do this, best thing is to create a new pull request from your repo, comment on it that it replaces this one, comment on this one that it is replaced by the new one, and then close this one. Then assign the new one to me.

This sounds complex, but it just makes tracking easier.

**Comment by pmuir** [comment]
Also, as always, WORK IN BRANCHES! :-D

**Comment by pmuir** [comment]
Merged.

I redid the merge and included a rebase.

**Pull request closed without merging**